src/BootstrapForm.php

```diff
@@ -36,6 +36,13 @@ class BootstrapForm
      * @var \Illuminate\Session\Store
      */
     protected $session;
+	
+    /**
+     * Set to true for horizontal form.
+     *
+     * @bool
+     */
+    protected $use_column_classes = false;
```

Comment:
Variable names should be camel case. Perhaps a more descriptive variable name would be `$horizontalForm`?